cartridges/CustomerPreferences/cartridge/forms/default/preferences.xml

```diff
@@ -1,37 +1,42 @@
 <?xml version="1.0"?>
 <form xmlns="http://www.demandware.com/xml/form/2008-04-19" secure="false">
 	<group formid="customerPreferences">
-		<field 
-			formid="birthday" 
-			label="label.profile.birthday" 
-			mandatory="false" 
-			max-length="50"
-			missing-error="error.message.required" 
-			range-error="error.message.50orless" 
+		<!-- Birhday -->
+		<field
+			formid="birthday"
+			label="label.profile.birthday"
+			mandatory="false"
+			missing-error="error.message.required"
+			range-error="error.message.50orless"
 			type="date" />
 
-		<field 
-			formid="interests" 
-			label="label.profile.interests" 
-			mandatory="true" 
-			max-length="50" 
+		<!-- Interests -->
+		<!-- Electronic -->
+		<field
+			formid="interests-electronic"
+			label="label.profile.interests"
```

Comment:
maybe change this to 'label.profile.interests.electronic' : electronic and 'label.profile.interests.apparel': apparel so we have 2 different labels